src/release-it-config/config.ts

```diff
@@ -0,0 +1,48 @@
+import {
+   IRepositoryReleaseSettings,
+   IReleaseItOptions,
+} from './interfaces';
+import path from 'path';
+
+const RELEASE_VERSION_NAME = 'release v${version}',
+      CHANGELOG_PATTERN: string | boolean = 'git log --pretty=format:"* %s (%h)" $(git describe --exclude "*rc*" --abbrev=0)...HEAD',
+      CHANGELOG_INFILE = 'CHANGELOG.md',
+      REPOSITORY_RELEASE_SETTINGS: IRepositoryReleaseSettings = {},
+      IS_DEV = (process.argv || []).indexOf('--dev') > -1;
+
+REPOSITORY_RELEASE_SETTINGS.release = true;
+REPOSITORY_RELEASE_SETTINGS.releaseName = 'Release ${tagName}';
+REPOSITORY_RELEASE_SETTINGS.releaseNotes = CHANGELOG_PATTERN;
+
+export default {
+   plugins: {
+      '@release-it/conventional-changelog': {
+         preset: 'conventionalcommits',
+         infile: CHANGELOG_INFILE,
+      },
+      // The `--dev` flag can be passed via the CLI to resolve the plugin at the
+      // correct location, otherwise when running inside the project, release-it
+      // will try to resolve it from `node_modules` instead of the project directory.
+      [
+      IS_DEV ?
+         path.resolve(__dirname) + '/plugins/pause-for-changelog.js' :
+         '@silvermine/standardization/dist/release-it-config/plugins/pause-for-changelog.js'
+      ]: {
+         infile: CHANGELOG_INFILE,
+      },
+   },
+   git: {
+      push: false,
+      tag: false,
+      tagName: 'v${version}',
+      tagAnnotation: RELEASE_VERSION_NAME,
+      commitMessage: 'chore: ' + RELEASE_VERSION_NAME,
+      changelog: CHANGELOG_PATTERN,
+      requireUpstream: false,
+   },
+   npm: {
+      publish: false,
+   },
+   gitHub: REPOSITORY_RELEASE_SETTINGS,
+   gitLab: REPOSITORY_RELEASE_SETTINGS,
```

Comment:
This doesn't seem to be outputting the correct URLs for Gitlab links in the CHANGELOG. Ping me for an IRL chat about what's happening.

Also, is there a way to configure the link pattern for ticket URLs?

Reply:
ADIRL this is because the project we were testing with didn't have a repository field in package.json. Will update docs to specify it's needed.